.github/workflows/test.yml

```diff
@@ -10,10 +10,8 @@ jobs:
     - uses: actions/checkout@v3
       name: Checkout project
 
-    - uses: leafo/gh-actions-lua@v9
```

Comment:
Seems like leafo/gh-actions-lua has some issues since a few months back. There's a PR to fix it but the author has not merged it. We might as well just install Lua using apt-get.